LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/RegionManager.cs

```diff
@@ -124,17 +123,28 @@ public static Region CN470
             {
                 if (cn470 == null)
                 {
-#pragma warning disable CA1508 // Avoid dead conditional code
-                    // False positive
-                    if (cn470 == null)
-#pragma warning restore CA1508 // Avoid dead conditional code
-                    {
-                        cn470 = new RegionCN470();
-                    }
+                    cn470 = new RegionCN470();
                 }
 
                 return cn470;
             }
         }
+
+        private static Region as923;
+
+        public static Region AS923
+        {
+            get
+            {
+                if (as923 == null)
+                {
+
+                    // Temporarily using these default parameter values until we can pass the configured frequencies to the region
```

Comment:
IS this coming in the next PR? 😀

Reply:
I discussed with @danigian and we added it in this PR in the `LnsStationConfiguration` as part of 7f02093d2ac949886017a1bc4d0172b05fbffd79. The default values have been removed from `RegionManager`.